Dfe.PrepareConversions/Dfe.PrepareConversions/Pages/TaskList/SchoolPupilForecasts/PupilForecasts.cshtml

```diff
@@ -1,4 +1,4 @@
-﻿@page "/task-list/{id:int}/confirm-school-pupil-forecasts"
```

Comment:
Why is this changing? And in the future we are adding the ability to edit the pupil forecast I believe, should this not indicate that this is a summary page? Same applies for budget etc

Reply:
Will it truncate or replace to slug, i.e., are you expecting budget/update-budget?